src/reducers/users.js

```diff
@@ -0,0 +1,40 @@
+let initialState = {
```

Comment:
I think this should be `users` instead of `initialState`.

Since the name is users.js. This file is related to all user data. So there will be no other data than users. So I think you don't have to wrap the users within `initialState`